mandatory/1-syntax-errors.js

```diff
@@ -1,17 +1,21 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-  return a + b + c;
+function addNumbers(a,b,c) {
+  return (a + b + c);
 }
 
-function introduceMe(name, age)
-  return `Hello, my {name}` is "and I am $age years old`;
+function introduceMe(name, age){
+  return (`Hello, my name is ${name} and I am ${age} years old`);}
 
 function getTotal(a, b) {
-  total = a ++ b;
+  total = (a + b);
 
-  return "The total is total";
+  return ('The total is ' + total);
```

Comment:
Your changes will work, but another way would have been to include the total inside the string - 
return (`The total is ${total}`);

Reply:
Don't forget the backticks though.
`return `The total is ${total}`;